lib/src/builders/voice.dart

```diff
@@ -32,16 +32,16 @@ class CurrentUserVoiceStateUpdateBuilder extends VoiceStateUpdateBuilder {
 class GatewayVoiceStateBuilder extends CreateBuilder<VoiceState> {
   Snowflake? channelId;
 
-  bool isMuted;
+  bool isSelfMuted;
 
-  bool isDeafened;
+  bool isSelfDeafened;
```

Comment:
Id rather we didn't rename these fields - isMuted and isDeafened are more consistent, and avoid a breaking change (even if we can make breaking changes before 6.0.0)

Reply:
Sure, I'll revert them back. 😄